### PR TITLE
update master branch script (and minor fixes to old branch script)

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
update master branch script (and minor fixes to old branch script)
new script goes through all the packs:

- delete any file whose "toversion" is lower than the given from version.
- update any "fromversion" to be the new from version (adds one if needed).

### CODE REVIEW

**Comment by roysagi** [comment]
Internal script - no docs needed

File: Utils/update_branch_from_version.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    click.secho(f"Starting process for {pack_path}:")
+    for content_dir in os.listdir(pack_path):
+        dir_path = os.path.join(pack_path, content_dir)
+        if content_dir in ['Playbooks', 'TestPlaybooks']:
+            edit_playbooks_directory(new_from_version, dir_path)
+
+        elif content_dir in ['Scripts', 'Integrations']:
+            edit_scripts_or_integrations_directory(new_from_version, dir_path)
+
+        elif content_dir in ['IncidentFields', 'IncidentTypes', 'IndicatorFields', 'Layouts', 'Classifiers',
```
Comment (ronykoz) [inline Utils/update_branch_from_version.py:203]: don't we have this list saved somewhere else?

File: Utils/update_branch_from_version.py [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+        with open(file_path, mode='w', encoding='utf-8') as f:
+            ryaml.dump(yml_content, f)
+            print(f" - Updating {file_path}")
+
+
+def check_dockerimage45(yml_content, new_from_version):
+    # check in scripts
+    if 'dockerimage45' in yml_content:
+        if parse_version(new_from_version) > parse_version('4.5.9'):
```
Comment (ronykoz) [inline Utils/update_branch_from_version.py:109]: let's make the version in a const

File: Utils/update_branch_from_version.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    print(f" - Deleting {file_path}")
+
+    changelog_file = os.path.join(os.path.splitext(file_path)[0] + '_CHANGELOG.md')
+    if os.path.isfile(changelog_file):
+        os.remove(changelog_file)
+
+
+def rewrite_json(file_path, json_content, new_from_version):
+    # if there is no fromVersion in json or the fromVersion is lower than the new from version then update
+    if ('fromVersion' in json_content and parse_version(json_content.get('fromVersion')) < parse_version(new_from_version)) \
```
Comment (ronykoz) [inline Utils/update_branch_from_version.py:71]: convert the condition to a function

**Review by ronykoz — APPROVED** [review]
Looks good

**Pull request merged**